### ISSUE TITLE
Make Tabris embeddable into third party native apps

### ISSUE THREAD

**Issue by mpost** [eclipsesource/tabris-js#1667]
### Problem description

In case a native app already exists on the customer side, it will be beneficial if a Tabris.js app could be embedded into that app. We need to determine if integrated as "fullscreen component" as a widget or in a headless mode.

How could such an app be build? With cordova? Likely without plugins.

### Environment

- Tabris.js version: 
- Device: 
- OS: 

### Code snippet

```javascript
// A minimal, *self-contained* code snippet to reproduce the problem:
```

**Comment by fax1ty** [comment]
Did it done yet? https://tabris.com/tabris-js-3-4-released/ (Possibly you can close it)

**Comment by mpost** [comment]
@fax1ty You are right. At the moment this only applies to Android though. I will refine the description text.
